list-components/AddTodo.js

```diff
@@ -0,0 +1,32 @@
+import React, { useState } from 'react';
+import { StyleSheet, Text, TextInput, Button, View } from 'react-native';
```

Comment:
```suggestion
import { StyleSheet, TextInput, Button, View } from 'react-native';
```
This file doesn't use Text.